packages/pwa-kit-runtime/src/ssr/server/test_fixtures/extensions/test-extension/setup-server.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+const extension = ({app}) => {
+    app.get('/test-extension', (req, res) => {
+        res.send('test')
+    })
+}
+
+module.exports = {
+    default: extension
+}
```

Comment:
Should we have tests for TS setup files. Also for coverage you might get better coverage if you have another test to see that the right thing happens when 2 setup files are created (one with js one with ts)

Reply:
Well, this `test_fixture` folder is actually the build artifact. Currently all of our tests skips Webpack... I'd like us to start adding tests that actually go thru webpack, but there are blockers and it's a large change so i want to propose that we do it in the future.

Reply:
I added ts setup files in the build-dev-server.test.js